### PR TITLE
update copy

### PR DESCRIPTION
**Pull request by rianfowler**
## Description
updated copy for this intro page 

## Testing done
verified locally 

## Testing Plan
validate screenshots 

## Screenshots
download

## Acceptance Criteria (Definition of Done)
sign off from UX or content 

#### Applies to all PRs

- [x] Provide link to originating GitHub issue, or connected to it via ZenHub
- [x] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)

### CODE REVIEW

**Comment by rianfowler** [comment]
@peggygannon @emilywaggoner I'm looking into why the alert banner is being displayed- [current](https://github.com/department-of-veterans-affairs/vets-website/pull/8150) logic has it always displaying when the application is up.

**Comment by cehsu** [comment]
LGTM, will leave it to @peggygannon to approve.

File: src/applications/post-911-gib-status/containers/IntroPage.jsx [hunk 1]
```diff
@@ -31,11 +31,21 @@ export class IntroPage extends React.Component {
...
             <Link id="viewGIBS" to="status" className="usa-button va-button-primary">View Your GI Bill Benefits</Link>
-            <h2>What if I have trouble accessing my benefit statement?</h2>
-            <p>If you're having trouble accessing the tool during the hours above, please try again. If it still doesn't work, please call the Vets.gov Help Desk at 1-855-574-7286.</p>
+            <p>Your Post-9/11 GI Bill Statement of Benefits might not be available if one of these is true:
+              <ul>
+                <li>The name on your Vets.gov account doesn’t exactly match the name we have in our Post-9/11 GI Bill records.</li>
+                <li>We’re still processing your education benefits application and we haven’t yet created a record for you. We usually process applications within 60 days. If you applied less than 60 days ago, please check back soon.</li>
+                <li>A family member or dependent is trying to access your benefit statement, and they might not be authorized to use the tool.</li>
+                <li>You haven’t yet applied for Post-9/11 GI Bill education benefits. <a href="/education/apply/">Apply for education benefits.</a></li>
```
Comment (cehsu) [inline src/applications/post-911-gib-status/containers/IntroPage.jsx]: Links should generally open in new tabs.

File: src/applications/post-911-gib-status/containers/IntroPage.jsx [hunk 2]
```diff
@@ -31,11 +31,21 @@ export class IntroPage extends React.Component {
...
+                <li>You’re not eligible for Post-9/11 GI Bill education benefits.</li>
+                <li> You’re trying to access the tool during its scheduled downtime. The tool is available Sunday through Friday, 6:00 a.m. to 10:00 p.m. (ET), and Saturday 6:00 a.m. to 7:00 p.m. (ET).</li>
```
Comment (cehsu) [inline src/applications/post-911-gib-status/containers/IntroPage.jsx]: Extra space at beginning of sentence.

**Review by cehsu — APPROVED** [review]

**Review by peggygannon — CHANGES_REQUESTED** [review]
@rianfowler I have just some slight tweaks to the message. Can you update the error message to: 

We're sorry. Something isn't working quite right with the GI Bill Benefits tool. We're working to fix the problem. If you're having trouble, please try again later.

**Comment by rianfowler** [comment]
@peggygannon updated the banner copy. Looks like this warning banner is going to stay a while until we have some time to meet with the stakeholders. 

download-1

**Review by peggygannon — APPROVED** [review]

**Comment by peggygannon** [comment]
Thanks, @rianfowler Updated screenshot looks good.

**Review by emilywaggoner — APPROVED** [review]
Looks good to me!

**Pull request merged**